carthage_command.sh

```diff
@@ -0,0 +1,14 @@
+xcodesdk=`xcodebuild -scheme Debug  -showBuildSettings  | grep -i 'SDK_VERSION =' | sed 's/[ ]*SDK_VERSION = //' | colrm 3`
```

Comment:
This workaround script is taken from Firefox

Reply:
fastlane runs Carthage as well, but uses the fastlane command directly… we may have to switch it to use this script instead?

Reply:
good point, it had to be done, I fixed it and tested by uploading dev build to app store